SubscribeIT/ngDesk
# Teams layout - users drop down selected user must be disabled

Closes #200 

Test Case 1
Name :- Teams create layout
1. Login to ngdesk.
2. Goto -> teams -> new.
3. Give name -> select user from drop down.
4. Observe the user is diabled.
5. Hit save.
6. Observe getting saved.
7. Observe no error.

Test Case 2
Name :- Teams edit layout
1. Login to ngdesk.
2. Goto -> teams -> click on teams created above.
3. Change name -> click on user field.
4. Observe the user is diabled.
5. Hit save.
6. Observe getting updated.
7. Observe no error.

Screenshots :- 

![Screenshot from 2021-09-21 16-45-33](https://user-images.githubusercontent.com/89504119/134161198-03dcd367-d6a9-4591-95fd-053472887320.png)
